Source/Core/DolphinQt/Config/GraphicsModListWidget.cpp

```diff
@@ -59,9 +63,10 @@ void GraphicsModListWidget::CreateWidgets()
   m_mod_list->setSelectionRectVisible(true);
   m_mod_list->setDragDropMode(QAbstractItemView::InternalMove);
 
+  m_open_directory_button = new NonDefaultQPushButton(tr("Open Directory..."));
```

Comment:
Is there a reason `NonDefaultQPushButton` was used here?

Reply:
I copied the implemention from the Resource pack manager which uses it. I thought it would be preferred: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinQt/ResourcePackManager.cpp#L38

Reply:
Theoretically `NonDefaultQPushButton` prevents the button from automatically being focused. If I change the one in `ResourcePackManager` I'm not experiencing it being automatically focused, though.

Since `GraphicsModListWidget` doesn't use `NonDefaultQPushButton` elsewhere it's probably better to stick with `QPushButton`, though.

Reply:
Thanks for the explanation @Pokechu22. No problem, I updated the code and it's all `QPushButton` now.

Just curious, was the idea some time ago to move everything to this custom `NonDefaultQPushButton`? Was the focus causing much trouble elsewhere in the settings?

Reply:
See #10497 and [issue 11184](https://bugs.dolphin-emu.org/issues/11184) - to my understanding game settings weren't affected, but this isn't something I know too much about.